subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -789,7 +938,12 @@ static int iso_handle(const struct device *dev, struct net_buf *buf)
 	err = hci_iso_handle(buf, &evt);
 	if (evt) {
 		LOG_DBG("Replying with event of %u bytes", evt->len);
-		bt_recv_prio(dev, evt);
+
+#if defined(CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE)
+		err = bt_recv_prio(dev, evt);
+#else /* CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE */
+		err = bt_recv(dev, evt);
+#endif /* CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE */
```

Comment:
Not neccesarrily something that needs an update, but `bt_recv` is only ever used in this combination. We could have `bt_recv` call `bt_recv_prio` depending on `CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE` so that this just becomes
```suggestion
		err = bt_recv(dev, evt);
```

Semantically that's not a good solution though I admit, given the naming of these functions

Reply:
`bt_recv` and `bt_recv_prio`  is more clear as to the priority of the function execution, lets keep it as-is.